obspy/taup/tau.py

```diff
@@ -663,3 +713,91 @@ def create_taup_model(model_name, output_dir, input_dir):
     else:
         model_file_name = model_name + ".tvel"
     TauPCreate.main(model_file_name, output_dir, input_dir)
+
+
+def traveltime_plot(min_degree=0, max_degree=360, npoints=1000,
+                    phases=['P', 'S'], source_depth=10, model='iasp91',
+                    ax=None, show=True):
+        """
+        Method to plot the travel times of arrivals class, if any have
+        been calculated.
+
+        :param min_degree: minimum distance from the source (in degrees) to 
+            plot travel times Defaults to ``0``.
+        :type min_degree: float, optional
```

Comment:
I think we nowadays usually leave away the `, optional` part (I know we still have it in older parts of the docs).

It's just plain redundant to add it for just every `kwarg` out there and only clutters the docs, IMHO.